samples/BoltOn.Samples.Infrastructure/Data/CollegeDbContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using BoltOn.Data.CosmosDb;
+using Microsoft.Extensions.Options;
+
+namespace BoltOn.Samples.Infrastructure.Data
+{
+    public class CollegeDbContext : BaseCosmosDbContext
+    {
+        //private readonly CosmosSettings _settings;
+        private const string DatabaseName = "College";
+
+        public CollegeDbContext(IOptions<CosmosDbSettings> settings) : base(settings.Value, DatabaseName)
+        {
+            // _settings = settings.Value;
+        }
+
+        protected override void SetCosmosDbSetting()
```

Comment:
u can make this method in the base class virtual, so that the inherited classes need not override when there is nothing to override